guidelines/index.html

```diff
@@ -3494,16 +3494,12 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    
 </dd>
 
-                <dt class="new"><dfn data-lt="pointer inputs|pointer input" data-dfn-type="dfn" id="dfn-pointer-inputs">ポインタ入力 (pointer input)</dfn></dt>
-<dd class="new">
-   					
-   
-   					
-   <p>マウス、ペン、タッチ接触のように、画面上の特定の (一つ又は複数の) 地点を指すことができる入力デバイス。
+                <dt><dfn data-lt="pointer inputs|pointer input" data-dfn-type="dfn" id="dfn-pointer-inputs">ポインタ入力 (pointer input)</dfn></dt>
+<dd>
+   <p>マウス、ペン、タッチ接触のように、画面上の特定の (1 つ又は複数の) 座標をターゲットにできるデバイスからの入力。
```

Comment:
https://github.com/waic/wcag21/pull/1846#issuecomment-1774587429 にも書きましたがこうかも?

```suggestion
   <p>マウス、ペン、タッチ接触のように、画面上の特定の座標 (又は複数の座標群) をターゲットにできるデバイスからの入力。
```